src/tree/CosmosDBAccountWizard/CosmosDBAccountCapacityStep.ts

```diff
@@ -23,16 +51,25 @@ export class CosmosDBAccountCapacityStep extends AzureWizardPromptStep<ICosmosDB
         let pick: IAzureQuickPickItem<boolean | undefined>;
 
         do {
-            pick = await context.ui.showQuickPick(picks, { placeHolder, suppressPersistence: true });
+            pick = await context.ui.showQuickPick(picks, {
+                placeHolder,
+                suppressPersistence: true,
+                learnMoreLink: learnMoreLink,
+            });
             if (pick === learnMore) {
-                await openUrl('https://aka.ms/cosmos-models');
+                await openUrl(learnMoreLink);
             }
         } while (pick === learnMore);
 
         if (pick.data) {
             context.isServerless = pick.data;
             context.telemetry.properties.isServerless = pick.data ? 'true' : 'false';
         }
+        if (pick === vcore) {
+            await openUrl("https://learn.microsoft.com/azure/cosmos-db/mongodb/vcore/quickstart-portal");
+            context.telemetry.properties.isvCore = 'true';
```

Comment:
I see isServerless, isvCore in here. why don't we add the experience property as well in here?